tests/test_groups.py

```diff
@@ -216,6 +216,27 @@ def test_groups_set_type(logged_rocket):
     assert not groups_set_type.get('success')
 
 
+def test_groups_set_custom_fields(logged_rocket, test_group_id, test_group_name,):
+    field_name = str(uuid.uuid1())
+    field_value = str(uuid.uuid1())
+    custom_fields = {field_name: field_value}
+
+    groups_set_custom_fields = logged_rocket.groups_set_custom_fields(
+        custom_fields, room_id=test_group_id).json()
+    assert groups_set_custom_fields.get('success')
+    assert groups_set_custom_fields.get('group').get('customFields') == custom_fields, \
+        'Custom fields do not match'
+
+    groups_set_custom_fields = logged_rocket.groups_set_custom_fields(
+        custom_fields, room_name=test_group_name).json()
+    assert groups_set_custom_fields.get('success')
+    assert groups_set_custom_fields.get('group').get('customFields') == custom_fields, \
+        'Custom fields do not match'
+
+    with pytest.raises(RocketMissingParamException):
+        logged_rocket.logged_rocket.groups_set_custom_fields(custom_fields)
```

Comment:
There's a repeated `logged_rocket.` here which is making the tests fail.
For some weird reason, it's not showing in the travis check but it is reflected in the coverage analysis.
Once this is corrected I'll merge your changes :)

Reply:
Durf, sorry about that.

Reply:
no prob ;)